doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
+| `req.inf.ntw.13` | Network | The Architecture **must** support dual stack IPv4 and IPv6 in Pods and Services. |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+
+<p align="center"><b>Table 2-2:</b> Containerised Platform: Infrastructure Requirements.</p>
+
+<a name="2.3.3"></a>
+### 2.3.3 Containerised Infrastructure Management (CIM)
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.cim.01` | General | The Architecture **must** allow infrastructure resource sharing. |
+| `req.cim.02` | General | The Architecture **must** allow discovery and management of NFVI resources. |
+| `req.cim.03` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
+| `req.cim.04` | General | The Architecture **must** include image repository management. |
+| `req.cim.05` | General | The Architecture **should** support a multi-tenanted environment. |
+| `req.cim.06` | General | The Architecture **must** support resource tagging. |
+| `req.cim.07` | General | The Architecture **must** support horizontal scaling. |
+| `req.cim.08` | General | The Architecture **must** support workload resiliency. |
+
+<p align="center"><b>Table 2-3:</b> Containerised Platform: CIM Requirements.</p>
+
+<a name="2.3.4"></a>
+### 2.3.4 Interfaces & APIs
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services. |
+| `req.int.api.02` | API | The Architecture **must** provide APIs needed to discover and manage NFVI resources. |
+
+
+<p align="center"><b>Table 2-4:</b> Containerised Platform: Interfaces and APIs Requirements. </p>
+
+<a name="2.3.5"></a>
+### 2.3.5 LCM
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.lcm.gen.01`	| General | The Architecture **must** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement). |
+| `req.lcm.adp.01` | Automated deployment | The Architecture **should** allow for “cookie cutter” automated deployment, configuration, provisioning and management of multiple NFVI sites. |
```

Comment:
Where is this coming from? There is nothing about multisite infra LCM in the RM. I'm not sure if we should define infra LCM in the RA at all.

Reply:
Where would you describe it? The RM does discusses it here:
https://github.com/cntt-n/CNTT/blob/pgoyal01-patch-4/doc/ref_model/chapters/chapter09.md

Moreover, I understand that - at least initially - there will be a need for multiple K8s clusters to make up for the lack of multi-tenancy capabilities.

Reply:
[RM Ch9](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter09.md) does not say anything about multisite LCM. 

Yes, in mult-tenancy what we will have I guess is a set of gaps and a strategy to work around them for example with one Kubernetes cluster per security domain. But these clusters are still in one site.

Reply:
Does it make a big difference if they are not? I mean if we have a central logic that can LCM multiple CaaS instances, it's a matter of network connectivity to be able to do it multi-site. And in the majority of the cases service providers will want to build things across multiple sites (for geographical network resilience). I do not insist on it, but I don't think it's such a big leap compared to multi-CaaS single-site.

Reply:
I think instead of "NFVI Sites" it should read "CaaS Clusters" or "Kubernetes Clusters" and then it's ok?  It can be single site, multi site, whatever and chapter 7 will address the requirements of the CaaS to enable that.

Reply:
Yes, absolutely. 

On a sidenote: I must admit, I'm a bit hesitant on "cookie cutter". I instinctively figured out what it does mean, but I would like to make it more... formal. Here is an attempt:

" The Architecture **should** allow for automated deployment, configuration, provisioning and management of multiple - declaratively specified -  CaaS clusters. "